src/jupyter-aou/build/snippets/All of Us Cromwell Setup Python snippets/(1) Setup.py

```diff
@@ -0,0 +1,130 @@
+import os
+import subprocess
+import json
+import requests
+from pathlib import Path
+
+def check_for_app(env):
+    list_apps_url = f'{env["leonardo_url"]}/api/google/v1/apps/{env["google_project"]}'
```

Comment:
how are the snippet used? this is not valid for us

Reply:
I just imported all of the ones they had before, I'm not sure which snippets they'll want to keep or change but I'm not worried about that for now.

I could also just add a sample snippet and import later as needed, would you prefer that?